README.md

```diff
@@ -515,3 +446,13 @@ Run `npm publish --workspaces` in root node-wot folder.
 1. Run `npm dedupe` (see https://github.com/eclipse-thingweb/node-wot/pull/765#issuecomment-1133772886)
 
 </details>
+
+## License
+
+Dual-licensed under both
```

Comment:
```suggestion
Dual-licensed under:
```

I have removed both since if someone does not read the sentence later on, they can think of it as an AND relationship